fanaSystem/fanaSystem/middlewares.py

```diff
@@ -0,0 +1,61 @@
+from django.urls import resolve, reverse
+from django.http import JsonResponse
+from .utils import generic_error_handler
+from jwt.exceptions import ExpiredSignatureError, InvalidTokenError
+from decouple import config
+import jwt
+from django.utils.deprecation import MiddlewareMixin
+from .error_handler import ErrorHandler
+
+
+
+class ErrorHandlingMiddleware(MiddlewareMixin):
+    def process_exception(self, request, exception):
+        """
+        This method is called when an exception is raised in the view.
+        It uses the ErrorHandler to return an appropriate response.
+        """
+        return ErrorHandler.handle_error(exception)
+
+
+
+from django.urls import resolve
+from django.http import JsonResponse
+from decouple import config
+import jwt
+from .error_handler import ErrorHandler
+
+class RouteSpecificMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # Define the allowed named routes
```

Comment:
Is it different then JWTAuthenticationMiddleware (fanaSystem\fanaDashboard\middleware.py)?

Reply:
If not then shift remove JWTAuthenticationMiddleware, to remove duplicacy